libs/pages/settings/src/lib/feature/page-organization-billing-summary-feature/show-usage-modal-feature/show-usage-modal-feature.tsx

```diff
@@ -0,0 +1,76 @@
+import { Organization, type OrganizationCurrentCost } from 'qovery-typescript-axios'
```

Comment:
```suggestion
import { type Organization, type OrganizationCurrentCost } from 'qovery-typescript-axios'
```
The CI of the PR should be good now!